src/noob/event.py

```diff
@@ -46,3 +46,12 @@ class MetaEvent(Event):
 
     node_id: Literal["meta"]
     signal: MetaEventType
+
+
+class NoEvent:
```

Comment:
there's got to be a better way to do a non-numeric const in python, serializing this will mean some custom serialization i think